README.md

```diff
@@ -104,14 +101,6 @@ When you are finished, please do go ahead and create a [pull request](https://he
 
 Hops is entirely written in ECMAScript 5 and its code is formatted using [prettier](https://prettier.io). Please make sure your contribution does, too.
 
-### Alternatives
-
-* [Razzle](https://github.com/jaredpalmer/razzle)
-* [Next](https://github.com/zeit/next.js/)
-* [Gatsby](https://github.com/gatsbyjs/gatsby)
-* [Backpack](https://github.com/palmerhq/backpack)
-* [Create React App](https://github.com/facebook/create-react-app)
-
```

Comment:
I wouldn't remove this list and instead maybe update it. There's also [After.js](https://github.com/jaredpalmer/after.js) now for example.

Reply:
We should compare to alternatives or remove IMHO. For now removing was easier :)